types/tx_test.go

```diff
@@ -52,105 +49,6 @@ func TestTxIndexByHash(t *testing.T) {
 	}
 }
 
-func TestValidTxProof(t *testing.T) {
```

Comment:
deleted this because I also deleted the old way to create proofs for txs. We can keep both though if we want.